lang/en/tool_mergeusers.php

```diff
@@ -194,3 +194,9 @@
 $string['finishtime'] = 'Finished merging at {$a}';
 $string['timetaken'] = 'Merge took {$a} seconds';
 $string['privacy:metadata'] = 'The Merge User Accounts plugin does not store any personal data.';
+$string['queue_to_process_pending_merge_requests'] = 'Get queue of the pending merge requests and launch adhoc task for each single request';
+$string['cannotfinduser'] = 'Cannot find the user with {$a->userfield} {$a->uservalue}.';
+$string['toomanyusers'] = 'More than one user found with {$a->userfield} {$a->uservalue}.';
+$string['maxattempts'] = 'Max number of attempts of a merge request.';
+$string['maxattempts_desc'] = 'This is the maximum number of attempts that a merge request will be tried to merge. After that number of attempts without success, the request will not be processed anymore and it will be updated as ended with errors.';
+$string['failedmergerequest'] = 'Failed merge request n° {$a->id}. Number of attempts: {$a->retries}';
```

Comment:
do not use `º`, instead use the word, in this case, `Failed merge request with id {$a->id}, on the attempt number {$a->retries}.`.

Reply:
done

Reply:
You left 2 spaces between `reqeust  with`. Leave just a single space.